### PR TITLE
Added pizza and wok to the bot's initial reactions

### PR DESCRIPTION
**Pull request by gijshendriksen**
Make sure the new pizza and wok options are automatically recommended by the bot

### CODE REVIEW

**Review by pingiun — COMMENTED** [review]
Maybe we can define this list using the filtering list for food. That way we only have to define the food list once

**Comment by gijshendriksen** [comment]
This was fixed in 6e6ef77. At least, I suppose something like this is what you mean?

File: bot.py [hunk 1]
```diff
@@ -141,22 +154,17 @@ def post_vote(bot, channel):
...
-        "Fest: :fries:\n"
-        "Appie: :ah:\n"
-        "Subway: :sandwich:\n"
-        "Pizza: :pizza:\n"
-        "Wok: :dragon_face:\n"
-        "I'm eating at home: :house:\n"
-        "I'm not going today: :x:"
+        "<!everyone> What do you want to eat today?\n" +
+        "\n".join([
+            f"{desc}: :{label}:" for (label, desc) in ALL_REACTIONS.items()
```
Comment (pingiun) [inline bot.py]: Good change overal, but these parenteces around label, desc are not necessary.

File: bot.py [hunk 2]
```diff
@@ -20,13 +20,26 @@
...
+    'ah': 'Appie',
+    'sandwich': 'Subway',
+    'pizza': 'Pizza',
+    'dragon_face': 'Wok',
+}
+HOME_REACTIONS = {
+    'house': "I'm eating at home",
+    'x': "I'm not going today",
+}
+ALL_REACTIONS = dict(EAT_REACTIONS, **HOME_REACTIONS)
```
Comment (pingiun) [inline bot.py]: Change this to `{**one, **two}` for bonus style points

File: bot.py (hunk 1 above)
Comment (thomwiggers) [inline bot.py]:
```suggestion
            f"{desc}: :{label}:" for label, desc in ALL_REACTIONS.items()
```

File: bot.py (hunk 2 above)
Comment (thomwiggers) [inline bot.py]:
```suggestion
ALL_REACTIONS = {**EAT_REACTIONS, **HOME_REACTIONS}
```

**Review by thomwiggers — COMMENTED** [review]
Converted @pingiun's remarks into suggested changes.

**Review by thomwiggers — APPROVED** [review]
Should be good now

**Pull request merged**